resources/views/community/components/forum/post.blade.php

```diff
@@ -32,7 +33,7 @@
     $commentDateModified = $commentData['DateModified'];
     $commentIsAuthorised = $commentData['Authorised'];
 
-    $isCurrentUserAdmin = $currentUserPermissions >= Permissions::Admin;
+    $isCurrentUserAdmin = $currentUserPermissions >= Permissions::Moderator;
```

Comment:
nit: Propose renaming this variable to `$isCurrentUserModerator`.

Reply:
Changed in https://github.com/RetroAchievements/RAWeb/pull/1772/commits/aecc9ba14dbecefab84b6a256e3e9eaeed420627